include/exadg/operators/multigrid_operator.h

```diff
@@ -150,6 +150,20 @@ class MultigridOperator : public MultigridOperatorBase<dim, Number>
   }
 #endif
 
+#ifdef DEAL_II_WITH_PETSC
+  virtual void
+  init_system_matrix(PETScWrappers::MPI::SparseMatrix & system_matrix) const
```

Comment:
```suggestion
  void
  init_system_matrix(PETScWrappers::MPI::SparseMatrix & system_matrix) const override
```
or depending on whatever #45 will settle on

Reply:
In my opinion, we should better separate the PRs in the future. Now we have different kinds of changes not only in the previous PR #45 , but also in the present one?

Wouldn't it make more sense to focus here on the AMG aspects, and consider the `virtual` things in the other PR #45, once we have decided there what to do?

Reply:
We can do that. Should we undo the commit?

Reply:
yes, I think it would be clearer to address these aspects in PR #45